scripts/build_sos_wv_database.py

```diff
@@ -49,6 +51,22 @@ def arguments_parser():
         metavar="FILE",
         help="File with filter terms, one per line",
     )
+
+    weaviate_args = parser.add_argument_group("Weaviate arguments")
+    weaviate_args.add_argument(
+        "--weaviate-host",
+        "--host",
+        type=str,
+        default="localhost",
+        help="Weaviate server host",
+    )
+    weaviate_args.add_argument(
+        "--weaviate-port",
+        "--port",
+        type=int,
+        default=8080,
```

Comment:
It's nice to be able to choose the port, we just need to clarify that this port should always match the Docker compose config file.